OpenStreetMapViewer/src/main/java/org/osmdroid/samplefragments/tileproviders/SampleVeryHighZoomLevel.java

```diff
@@ -0,0 +1,58 @@
+package org.osmdroid.samplefragments.tileproviders;
+
+import org.osmdroid.samplefragments.BaseSampleFragment;
+import org.osmdroid.tileprovider.MapTileProviderArray;
+import org.osmdroid.tileprovider.modules.MapTileApproximater;
+import org.osmdroid.tileprovider.modules.MapTileAssetsProvider;
+import org.osmdroid.tileprovider.modules.MapTileModuleProviderBase;
+import org.osmdroid.tileprovider.tilesource.ITileSource;
+import org.osmdroid.tileprovider.tilesource.XYTileSource;
+import org.osmdroid.tileprovider.util.SimpleRegisterReceiver;
+import org.osmdroid.util.GeoPoint;
+import org.osmdroid.views.overlay.ScaleBarOverlay;
+
+/**
+ * A lousy example of very high zoom levels.
```

Comment:
This is brilliant

Reply:
Thank you! I like it when I just click on the zoom out button, from level 29 down to 0.
Btw if you have access to "less abstract" tiles up to zoom level 29, or if you feel like adding polygons / polylines as examples, don't hesitate to enhance this demo.